examples/basics/export_data.ipynb

```diff
@@ -31,7 +31,7 @@
       "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    PROJECT_ID = "clpu7vm3w07uc07xf2aoe8rk4"
</pre><p>Could you please also remove this variable. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>